apisix/core/id.lua

```diff
@@ -65,9 +66,15 @@ function _M.init()
         return
     end
 
-    uuid.seed()
-    apisix_uid = uuid.generate_v4()
-    log.notice("not found apisix uid, generate a new one: ", apisix_uid)
+    --allow user to specify a meaningful id as apisix instance id
+    local local_conf = fetch_local_conf()
+    if local_conf and local_conf.apisix and local_conf.apisix.id then
```

Comment:
Can we use try_read_attr?
https://github.com/apache/apisix/blob/ce1d40b42f056f46b20386e1937fc81ba69e84c6/apisix/core/table.lua#L63

Reply:
solve it